client/src/components/generic/AboutSection/AboutSection.tsx

```diff
@@ -24,29 +24,25 @@ const TextStyler = ({ title, text }: Props) => {
 const AboutSection = ({ title, text, imageSrc, variant }: Props) => {
   if (variant === "left") {
     return (
-      <div className="relative left-0 w-[71%]">
-        <img src={imageSrc} className="rounded-tr-lg"></img>
-        <div className=" border-dark-blue-100 absolute top-[100%] flex w-full flex-col overflow-hidden rounded-b-lg border bg-white md:left-[72%] md:top-[70%] md:h-[60%] md:w-[50%]  md:rounded-t-lg lg:left-[35%] lg:top-[70%] lg:h-[56%] lg:w-[35%] lg:gap-4  lg:rounded-t-lg">
-          <TextStyler
-            title={title}
-            text={text}
-            variant={"left"}
-            imageSrc={imageSrc}
-          />
+      <div className="w-full">
+        <img
+          src={imageSrc}
+          className="relative w-full rounded-t-lg md:h-[500px] md:w-[900px]"
```

Comment:
your images are stretching and changing aspect ratio, look into https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit `object-fit: cover`